### DISCUSSION TITLE
[parser-packet-length] - Support multi-byte delimiter

### DISCUSSION THREAD

**Discussion by alferz** [serialport/node-serialport#2679] [Ideas]
It would be awesome if this parser could support a multiple byte delimiter. I have run into times where a protocol defines a 2-byte start sequence, such as 0xAA 0x43. In the current implementation I can use 0xAA but specifying an additional level of specificity would help cut down on possible errors. Could the 'delimiter' property accept a Buffer in addition to a number?

**Comment by bbeck13** [comment]
I've done this here https://github.com/serialport/node-serialport/issues/2709 I've also added support for multiple delimiters
